pkgs/tools/audio/goxlr-utility/default.nix

```
@@ -0,0 +1,65 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, installShellFiles
+, pkg-config
+, libpulseaudio
+, dbus
+, speechd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "goxlr-utility";
+  version = "v0.12.6";
```

Oh one small thing I haven't noticed before:

```suggestion
  version = "0.12.6";